js/main.js

```diff
@@ -0,0 +1,115 @@
+
+const PHOTOS_DESCRIPTION = [
+  'пляж',
+  'указатель к пляжу',
+  'море',
+  'девушка в купальнике',
+  'рисовые человечки',
+  'спорткар',
+  'завтрак из клубники',
+  'брусничный морс',
+  'люди машут самолёту',
+  'шкафчик для обуви',
+  'вход на пляж',
+  'автомобиль ауди',
+  'вкусный салат',
+  'кото-ролл',
+  'робо-тапки',
+  'вид из окна самолёта',
+  'выступление хора',
+  'раритетное авто',
+  'тапки с подсветкой',
+  'внутренний двор отеля',
+  'корейская еда',
+  'закат на море',
+  'краб',
+  'большой концерт',
+  'сафари с бегемотом',
+];
+
+const MESSAGE_TEXT_PARAGRAPH = [
+  'Всё отлично!',
+  'В целом всё неплохо. Но не всё.',
+  'Когда вы делаете фотографию, хорошо бы убирать палец из кадра.В конце концов это просто непрофессионально.',
+  'Моя бабушка случайно чихнула с фотоаппаратом в руках и у неё получилась фотография лучше.',
+  'Я поскользнулся на банановой кожуре и уронил фотоаппарат на кота и у меня получилась фотография лучше.',
+  'Лица у людей на фотке перекошены, как будто их избивают.',
+  'Как можно было поймать такой неудачный момент?!'
+];
+
+const USER_NAME = [
+  'Фёдор',
+  'Виктория',
+  'Герман',
+  'Илья',
+  'Тимофей',
+  'Виктория'
+];
+
+const createDescriptionCount = 25;
+
+
+const getRandomInteger = (min, max) => {
+  const lower = Math.ceil(Math.min(Math.abs(min), Math.abs(max)));
+  const upper = Math.floor(Math.max(Math.abs(min), Math.abs(max)));
+  const result = Math.random() * (upper - lower + 1) + lower;
+
+  return Math.floor(result);
+};
+
+const createIdGeneratorRandom = (min, max) => {
+  const previousValues = [];
+
+  return function () {
+    let currentValue = getRandomInteger(min, max);
+    if (previousValues.length >= (max - min + 1)) {
+      return null;
+    }
+    while (previousValues.includes(currentValue)) {
+      currentValue = getRandomInteger(min, max);
+    }
+    previousValues.push(currentValue);
+    return currentValue;
+  };
+};
+
+const createIdGenerator = () => {
+  let lastGeneratedId = 0;
+
+  return function () {
+    lastGeneratedId += 1;
+    return lastGeneratedId;
+  };
+};
+
+const generateCommentId = createIdGenerator();
+
+const createComments = () => (
+  {
+    id: generateCommentId(),
+    avatar: `img/ ${getRandomInteger(1, 6)} .svg`,
```

Comment:
```suggestion
    avatar: `img/${getRandomInteger(1, 6)}.svg`,
```
в бэктиках пробелы не нужно ставить, иначе пробелы будут в итоговой строке.
6 - магическое число